### PR TITLE
add Narik svg

### PR DESCRIPTION
**Pull request by abadakhshan**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -192,6 +192,13 @@
       "name": "Angular Thailand",
       "description": "Logo of the Angular Developers Thailand community",
       "license": "MIT"
+    },
+    "narik.svg": {
+      "creator": "https://github.com/NarikMe",
+      "name": "Narik",
+      "description": "Logo of the Angular Developers Thailand community",
```
Comment (fmalcher) [inline metadata.json]: I think this line is not correct ;-) @abadakhshan

Comment (abadakhshan) [reply metadata.json]: Thank you  @fmalcher 
I fixed it.

**Comment by abadakhshan** [comment]
Hello @fmalcher.  Is there any problem with this PR?

**Comment by fmalcher** [comment]
Thanks for your patience @abadakhshan , having a busy time right now. 🙂
From my side there is no problem with this PR, but merging it will be up to @maartentibau .
My only concern is that the Narik logo is not very close to the Angular logo which has 6 instead of 8 corners. What do you think? @maartentibau

**Pull request closed without merging**